internal/controller/kubecostextractor_controller.go

```diff
@@ -69,7 +73,7 @@ func (r *KubecostExtractorReconciler) RunOnInterval(ctx context.Context, key str
 	r.Tasks.Set(key, cancel)
 
 	go func() {
-		_ = wait.PollUntilContextCancel(ctxCancel, interval, true, condition)
+		_ = wait.PollUntilContextCancel(ctxCancel, interval+time.Duration(rand.Int63n(int64(kubeCostJitter))), true, condition)
```

Comment:
I don't think this will fully work because it will keep the jitter constant through the poll loop.  I'd rather us sleep for the jitter amount at the top of the loop, making it random each run.

Reply:
yes, you are right, I will change it

Reply:
done